### DISCUSSION TITLE
How to disable reading-time2 plugin?

### DISCUSSION THREAD

**Discussion by narumi147** [vuepress-theme-hope/vuepress-theme-hope#1080] [Q&A] [answered]
On the top of each normal page, there are Author+reading time info. I can hide author by omitting `themeConfig.author`, any idea to hide reading time? Thanks.

I'm using v2 theme.

**Comment by Mister-Hope** [comment, marked as answer]
Emmm, v2 theme is not finished. Only plugins are available now. So I won't work on any bug report about theme(will handle them in future).

But gladly, the reading time is displayed by "PageInfo" provided by comment plugin. You may need to check the v1 docs(changes are all mentioned in readme, and there are only a few changes)

**Comment by narumi147** [comment]
Thanks, I have switched v1 now.

For v2, I once wanted to disable comment plugin with `type: "disable"` but vscode keeps showing the type error. And just now tested that adding `pageInfo: false` does work to hide reading time.

**Reply by Mister-Hope** [reply]
Thanks for the detailed information, I will have a look.

P.S.: Remember, just as sidebar in official theme, not all the features support hot reload. So make sure you restart the project after changing it.
